source/renderer/app/components/wallet/settings/WalletSettings.js

```diff
@@ -115,101 +117,127 @@ export default class WalletSettings extends Component<Props> {
     } = this.props;
 
     return (
-      <div className={styles.component}>
-        <BorderedBox>
-          <InlineEditingInput
-            className="walletName"
-            inputFieldLabel={intl.formatMessage(messages.name)}
-            inputFieldValue={walletName}
-            isActive={activeField === 'name'}
-            onStartEditing={() => onStartEditing('name')}
-            onStopEditing={onStopEditing}
-            onCancelEditing={onCancelEditing}
-            onSubmit={value => onFieldValueChange('name', value)}
-            isValid={nameValidator}
-            validationErrorMessage={intl.formatMessage(
-              globalMessages.invalidWalletName
-            )}
-            successfullyUpdated={
-              !isSubmitting && lastUpdatedField === 'name' && !isInvalid
-            }
-          />
-
-          <ReadOnlyInput
-            label={intl.formatMessage(messages.passwordLabel)}
-            value={intl.formatMessage(messages.passwordLastUpdated, {
-              lastUpdated: moment(spendingPasswordUpdateDate).fromNow(),
-            })}
-            onClick={() =>
-              openDialogAction({
-                dialog: ChangeSpendingPasswordDialog,
-              })
-            }
-          />
-
-          <WalletRecoveryPhrase
-            recoveryPhraseVerificationDate={recoveryPhraseVerificationDate}
-            recoveryPhraseVerificationStatus={recoveryPhraseVerificationStatus}
-            recoveryPhraseVerificationStatusType={
-              recoveryPhraseVerificationStatusType
-            }
-            creationDate={creationDate}
-            openDialogAction={openDialogAction}
-            isDialogOpen={isDialogOpen}
-            walletRecoveryPhraseStep1Container={
-              walletRecoveryPhraseStep1Container
-            }
-            walletRecoveryPhraseStep2Container={
-              walletRecoveryPhraseStep2Container
-            }
-            walletRecoveryPhraseStep3Container={
-              walletRecoveryPhraseStep3Container
-            }
-            walletRecoveryPhraseStep4Container={
-              walletRecoveryPhraseStep4Container
-            }
-          />
-
-          {error && <p className={styles.error}>{intl.formatMessage(error)}</p>}
-
-          <div className={styles.actionButtons}>
-            {showExportLink ? (
-              <button
-                className={styles.exportLink}
+      (isLegacy && (
+        <div className={styles.component}>
+          <BorderedBox>
+            <h2 className={styles.deleteWalletLabel}>Delete Wallet</h2>
```

Comment:
@DeeJayElly please leave just the old styled "Delete Wallet" link here.
We will replace it with this new layout once we merge latest develop branch.

Reply:
Fixed